onvm/onvm_mgr/onvm_nf.c

```diff
@@ -86,6 +87,9 @@ onvm_nf_ready(struct onvm_nf_info *nf_info);
 inline static int
 onvm_nf_stop(struct onvm_nf_info *nf_info);
 
+static void
+init_lpm_region(struct lpm_request *req_lpm);
```

Comment:
+1 on @koolzz comments; please add comment in .h file describing init_lpm_region and change signature to follow our semantic